server/handlers/api-browsers.js

```diff
@@ -1,16 +1,13 @@
 import { URL } from 'node:url'
 
-import getBrowsers, {
-  QUERY_DEFAULTS,
-  REGION_GLOBAL
-} from '../lib/get-browsers.js'
 import { sendResponseAPI } from '../lib/send-response.js'
+import { getBrowsers } from '../lib/get-browsers.js'
 
-export default async function handleAPIBrowsers(req, res) {
+export async function handleAPIBrowsers(req, res) {
   let { searchParams: params } = new URL(req.url, `http://${req.headers.host}/`)
```

Comment:
This duplicates `new URL` in `server/index.js`.
Let's pass the `url`, as for the `handleMain(req, res, url)`.

And we are ready for a merge :)

Reply:
Fixed e01b5a2